examples/inference/run_llama_inference.py

```diff
@@ -1,98 +0,0 @@
-import argparse
```

Comment:
Is this file no longer used, or it's an unexpected removal caused by merging?

Reply:
I can't find this file in my branch

Reply:
Gotcha. I guess this will be fixed when you rebase `feature/colossal-infer`